SimplyPluralApi.yml

```diff
@@ -1633,52 +1690,87 @@ paths:
       parameters:
         - in: path
           name: id
+          required: true
+          description: Id of the channel, this can be a mongo Id but also a plain string (backwards-compatibility).
           schema:
             type: string
+        - in: query
+          name: limit
           required: true
-          description: String id of the channel to get messages from
+          description: Amount of messages to fetch
+          schema:
+            type: integer
+            minimum: 1
+            maximum: 100
+      tags:
+        - Chat Messages
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "./SimplyPluralApiBodies.yml#/components/schemas/chatMessageContainer"
+        "400":
+          description: BAD REQUEST
+        "404":
+          description: NOT FOUND
+
+  /v1/chat/category:
```

Comment:
Don't create seperate endpoints for one and the same endpoint with an optional id.